pkg/scheduler/scheduler.go

```diff
@@ -96,6 +97,8 @@ type schedulerOptions struct {
 	profiles                   []schedulerapi.KubeSchedulerProfile
 	extenders                  []schedulerapi.Extender
 	frameworkCapturer          FrameworkCapturer
+	// Set the parallel thread count for parallism package
```

Comment:
remove this field

Reply:
You want me to remove the `parallelism` field ? Would that disrupt the rest of the logic ?